packages/expect/src/__tests__/matchers.test.js

```diff
@@ -446,6 +484,17 @@ describe('.toEqual()', () => {
     });
   });
 
+  if (isBigIntAllowed) {
+    [[BigInt(1), BigInt(2)], [BigInt(1), 2]].forEach(([a, b]) => {
```

Comment:
Does it make sense to replace second test case with `[BigInt(1), 1]` to emphasize as specification that bigint and corresponding number are **not** deep equal?